static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -715,61 +715,327 @@ $navigation-horizontal-padding: $gs-gutter * 2 + $gs-gutter/2;
-        // Optically aligns the icon in it's container
-        margin-top: 5px;
-        margin-left: 4px;
+        &:after {
+            content: none;
+        }
+    }
+}
+
+// Scrolly variant
+.tertiary__variant-three {
+    .tertiary-navigation__item:last-of-type .tertiary-navigation__link {
+        margin-right: $gutter-small + $veggie-burger-small;
+
+        @include mq($mobile-medium) {
+            margin-right: $gutter-medium + $veggie-burger-medium - $gs-gutter/2;
+        }
+
+        @include mq(mobileLandscape) {
+            margin-right: $gutter-large + $veggie-burger-medium - $gs-gutter/2;
+        }
+    }
+}
+
+// Stacked variant
+.tertiary__variant-four {
+    .tertiary-navigation__item:last-of-type .tertiary-navigation__link {
+        &:after {
+            content: '';
+        }
+    }
+}
+
+/****************
+ * Changes to an Article Body
+ ****************/
+
+$caption-button-size: 32px;
+.has-new-header {
+
+    // When the image is at the top of the article, the headline and byline are moved into a div with margins. This counteracts that
+    // TODO: If this change is kept after the test the template will have to be rewriten and these overrides won't be needed
+    .content__head {
+        @include mq($until: mobileLandscape) {
+            margin-right: -$gs-gutter /2;
+            margin-left: -$gs-gutter /2;
+        }
+
+        @include mq($from: mobileLandscape, $until: phablet) {
+            margin-right: -$gs-gutter;
+            margin-left: -$gs-gutter;
+        }
+    }
+
+    @include mq($from: phablet, $until: tablet) {
+        .media-primary,
+        .content__head {
+            margin-left: -$gs-gutter;
+            margin-right: -$gs-gutter;
+        }
+    }
+
+    @include mq($until: tablet) {
+        .content__headline {
+            padding-top: $gs-baseline/3;
+        }
+
+        .inline-expand-image,
+        .content__labels {
+            display: none;
+        }
+
+        .reveal-caption {
+            position: absolute;
+            right: $gs-gutter/4;
+            width: $caption-button-size;
+            height: $caption-button-size;
+            z-index: 1;
+            background-color: transparent;
+            border-radius: 50%;
+        }
+        .reveal-caption--img {
+            bottom: $gs-baseline/2;
+        }
+
+        .reveal-caption-icon__svg {
+            position: absolute;
+            top: 0;
+            right: 0;
+            bottom: 0;
+            left: 0;
+            margin: auto;
+            filter: drop-shadow(0 0 $gs-baseline/2 rgba(0, 0, 0, .4));
+        }
+
+        .inline-camera .reveal-caption-icon__svg {
+            // Optical alignment
+            top: -2px;
+        }
+
+        .caption--main.caption--video {
+            padding-bottom: 0;
+        }
+
+        .caption--main.caption--img {
+            position: absolute;
+            background: rgba($multimedia-support-5, .8);
+            color: white;
```

Comment:
Thanks for that! I fixed this and a few other css bits here: https://github.com/guardian/frontend/pull/15097/commits/b1245b1ab1de36cef8fe9d842fcd3f99bb720b89